qa/checkstyle.xml

```diff
@@ -40,7 +40,7 @@
         </module>
         <module name="NeedBraces"/>
         <module name="LeftCurly">
-            <property name="maxLineLength" value="100"/>
+<!--            <property name="maxLineLength" value="100"/>-->
```

Comment:
please leave the comment, about the reason why this line was commented (reason was described in the migration document)

Reply:
done